deployment/7_upgrade_1_6.js

```diff
@@ -152,21 +152,6 @@ const deploy = async (network) => {
       [wrapper.contractAddress, utils.asciiToBytes32(wrapper._contract.contractName)]);
   }
 
-  // ////////////////////////////////////////////////////
-  // Disable KyberNetwork on the TokenPriceProvider
-  // ////////////////////////////////////////////////////
-
-  if (!await TokenPriceProviderWrapper.contract.managers(deploymentWallet.address)) {
-    const TokenPriceProviderAddManagerTx = await TokenPriceProviderWrapper.contract.addManager(deploymentWallet.address, { gasPrice });
-    await TokenPriceProviderWrapper.verboseWaitForTransaction(TokenPriceProviderAddManagerTx,
-      `Set ${deploymentWallet.address} as the manager of the TokenPriceProvider`);
-  }
-
-  const TokenPriceProviderSetKyberNetworkTx = await TokenPriceProviderWrapper.contract.setKyberNetwork("0x0000000000000000000000000000000000000000",
-    { gasPrice });
-  await TokenPriceProviderWrapper.verboseWaitForTransaction(TokenPriceProviderSetKyberNetworkTx,
-    "Disable the KyberNetwork on the TokenPriceProvider");
-
```

Comment:
Does it make sense to update this file? Since this PR is part of version 2.0 this file should just be removed.

Reply:
I've kept it as a reference but had to remove the obsoleted calls now. Might have a dedicated PR to do the deployment scripts for 2.0 as that would be sufficiently big a change to upgrade everything.